src/renderer/components/MessageWrapper.js

```diff
@@ -241,16 +242,19 @@ function convertMessageStatus (s) {
 }
 
 function convertContentType (message) {
-  var filemime = message.filemime
-  if (!filemime) return 'image/jpg'
+  const filemime = message.filemime
+  if (!filemime) return 'application/octet-stream'
   if (filemime === 'application/octet-stream') {
     switch (message.msg.viewType) {
       case C.DC_MSG_IMAGE:
         return 'image/jpg'
       case C.DC_MSG_VOICE:
         return 'audio/ogg'
+      case C.DC_MSG_FILE:
+        const type = mime.lookup(message.msg.file)
+        if (type) return type
```

Comment:
Lookup mime type using file name. For `.webm` files this will be `video/webm`. If `type` can't be fount it will be `false`, which then will fall through to `application/octet-stream` below.